examples/applications/svm_gui.py

```diff
@@ -319,6 +319,7 @@ def get_parser():
 
 def main(argv):
     op = get_parser()
+
```

Comment:
As per @amueller's review above, since this was working fine in IPython without the change, so we chose not to add unused code.